.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+* Author (your GitHub user):
+* Issue reference: 
+
+To Do:
+
+- [ ] Linter errors resolved?
```

Comment:
I don't quite understand this hint, please elaborate a little bit more...

Reply:
Hey @leonel-lordford. This adds a [basic template](https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) to better describe the PRs and their content.